st2api/st2api/controllers/v1/packs.py

```diff
@@ -284,6 +289,10 @@ class PacksController(BasePacksController):
     views = PackViewsController()
     index = PacksIndexController()
 
+    def __init__(self):
+        super(PacksController, self).__init__()
+        self.get_one_db_method = self._get_by_ref_or_id
```

Comment:
Yes, one option would be to pass the method name / callable to the decorator or similar, but I decided to go with current approach because it's more generic.

No matter what approach we go with, we will need to specify this information (how to retrieve the item by PK) in some form, and the current approach is that it's specified on the class as a class / instance variable.